docs/en/ingest-management/release-notes/release-notes-8.5.asciidoc

```diff
@@ -13,13 +14,45 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-8.5.1>>
 * <<release-notes-8.5.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.5.1 relnotes
+
+[[release-notes-8.5.1]]
+== {fleet} and {agent} 8.5.1
+
+Review important information about the {fleet} and {agent} 8.5.1 release.
+
+[discrete]
+[[enhancements-8.5.1]]
+=== Enhancements
+
+{agent}::
+* Improve shutdown logs {agent-pull}1618[#1618]
```

Comment:
This does not seem like a "new feature" so I've put it under enhancements.

Reply:
Agreed, it makes more sense here.